src/media/local-stream.ts

```diff
@@ -9,12 +10,30 @@ interface LocalStreamEvents {
   [LocalStreamEventNames.ConstraintsChange]: TypedEvent<() => void>;
 }
 
+export type TrackEffect = BaseEffect;
```

Comment:
I think we can just use the name web-media-effects defines (BaseEffect).  If anything, maybe web-media-effects should rename BaseEffect to TrackEffect?